weather_mv/setup.py

```diff
@@ -11,8 +11,19 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""Setup weather-mv.
 
-from setuptools import setup, find_packages
+This setup.py script makes use of Apache Beam's recommended way to install non-python dependencies to worker images.
+This is employed in order for a portable installation of cfgrib, which requires ecCodes.
```

Comment:
'in order for a ' sounds awkward.
Maybe 'employed to enable a'?

Reply:
I appreciate that, will revise.